src/fr/validation-inline.php

```diff
@@ -115,67 +103,12 @@
     'string'               => 'Ce champ doit être une chaîne de caractères.',
     'timezone'             => 'Ce champ doit être un fuseau horaire valide.',
     'unique'               => 'La valeur est déjà utilisée.',
-    'uploaded'             => "Le fichier n'a pu être téléversé.",
-    'url'                  => "Le format de l'URL n'est pas valide.",
+    'uploaded'             => 'Le fichier n\'a pu être téléversé.',
+    'url'                  => 'Le format de l\'URL n\'est pas valide.',
     'uuid'                 => 'Ce champ doit être un UUID valide',
-
-    /*
-    |--------------------------------------------------------------------------
-    | Custom Validation Language Lines
-    |--------------------------------------------------------------------------
-    |
-    | Here you may specify custom validation messages for attributes using the
-    | convention "attribute.rule" to name the lines. This makes it quick to
-    | specify a specific custom language line for a given attribute rule.
-    |
-    */
-
-    'custom' => [
+    'custom'               => [
         'attribute-name' => [
             'rule-name' => 'custom-message',
         ],
     ],
-
-    /*
-    |--------------------------------------------------------------------------
-    | Custom Validation Attributes
-    |--------------------------------------------------------------------------
-    |
-    | The following language lines are used to swap attribute place-holders
-    | with something more reader friendly such as E-Mail Address instead
-    | of "email". This simply helps us make messages a little cleaner.
-    |
-    */
-
-    'attributes' => [
```

Comment:
You delete attributes ?

Reply:
No, it has been moved alphabetically.

Reply:
I figured out how to apply the correct sorting. I will do it in the coming days.